ketcher-autotests/tests/utils/canvas/helpers.ts

```diff
@@ -81,16 +81,19 @@ export async function takeEditorScreenshot(
   options?: { masks?: Locator[] },
 ) {
   const editor = page.locator('[class*="App-module_canvas"]');
+  await delay(DELAY_IN_SECONDS.THREE);
```

Comment:
are you sure, that it's a good idea put 3 sec delay here? we have plenty tests which use this method, it increase time of passing all the tests significantly.

Reply:
We decided to build the delay directly into the screenshot function

Reply:
and i removed in the delay code. where possible